app/assets/stylesheets/_flashes.scss

```diff
@@ -0,0 +1,30 @@
+.flashes {
+  .flash-alert,
+  .flash-error,
+  .flash-notice,
+  .flash-success {
+    border-radius: 4px;
+    margin-bottom: 1rem;
+    padding: 1rem;
+  }
+
+  .flash-alert {
+    // yellow
+    background: #fff3cd;
```

Comment:
What do you think about putting these colors in `app/assets/stylesheets/_variables.scss`?